SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/ErrorAttachmentLog.cs

```diff
@@ -10,9 +10,11 @@ namespace Microsoft.AppCenter.Crashes
     /// <summary>
     /// Error attachment log.
     /// </summary>
-    [JsonObject("errorAttachment")]
+    [JsonObject(JsonIdentifier)]
```

Comment:
Do we really need a variable for this one?

Reply:
Yeah it's accessed in the Crashes class. This is the pattern used for all logs.